packages/build/src/plugins_core/blobs_upload/index.ts

```diff
@@ -6,30 +6,37 @@ import semver from 'semver'
 
 import { log, logError } from '../../log/logger.js'
 import { anyBlobsToUpload, getBlobsDir } from '../../utils/blobs.js'
+import { CoreStep, CoreStepCondition, CoreStepFunction } from '../types.js'
 
 import { getKeysToUpload, getFileWithMetadata } from './utils.js'
 
-const coreStep = async function ({
+const coreStep: CoreStepFunction = async function ({
   debug,
   logs,
   deployId,
   buildDir,
   quiet,
-  constants: { PUBLISH_DIR, SITE_ID, NETLIFY_API_TOKEN, API_URL },
+  packagePath,
+  constants: { SITE_ID, NETLIFY_API_TOKEN, NETLIFY_API_HOST },
 }) {
+  // This should never happen due to the condition check
+  if (!deployId || !NETLIFY_API_TOKEN || !NETLIFY_API_HOST) {
+    return {}
+  }
+
   const storeOpts: { siteID: string; deployID: string; token: string; apiURL: string; fetch?: any } = {
     siteID: SITE_ID,
     deployID: deployId,
     token: NETLIFY_API_TOKEN,
-    apiURL: API_URL,
+    apiURL: NETLIFY_API_HOST,
```

Comment:
Why are we changing the constant we use?

Reply:
I never found any reference of `API_URL` this seems to be not a constant we support

Reply:
Interesting! I was intrigued when I saw it in the original PR, but just assumed it existed and I didn't know about it. Maybe @kitop can shed some light.

As for using `NETLIFY_API_HOST`, I _think_ it only gives you the hostname part of the URL, so I believe you need to construct the URL from it like `https://${constants.NETLIFY_API_HOST}`?

Reply:
At least I could not find any reference of this in the whole repo or in the CLI. The NETLIFY_API_HOST is set to `api.netlify.com` which is the value we need. (adding the https part to it)